.github/workflows/ci.yml

```diff
@@ -62,12 +51,19 @@ jobs:
             cp config/application.example.yml config/application.yml
             cp config/database.example.yml config/database.yml
             bin/rails db:migrate RAILS_ENV=test
-      
+        
+      - name: Cache Webpack cache
+        id: cache-webpack
+        uses: actions/cache@v3
+        with:
+          path: node_modules/.cache/
+          key: ${{ runner.os }}-webpack
+          
       #Testing the pushed code onwards
       
       - name: JavaScript test suite 
         run: |
-            yarn
+            yarn install --immutable
```

Comment:
What is the `immutable` flag for here, and should we also be using that for building production assets for deployment, to make sure they match what is being tested in CI?

Reply:
`yarn install --immutable` fails if after installing the packages the lockfile changed. This happens when someone added a package to the `package.json` but forgot to commit the lockfile.